go/worker/keymanager/watcher.go

```diff
@@ -293,6 +319,11 @@ func (w *rtNodeWatcher) watch(ctx context.Context) {
 				watcher.Reset()
 				defer watcher.Freeze(0)
 
+				for sig := range observerNodes {
+					_, _ = watcher.WatchNode(ctx, sig)
```

Comment:
I understand that we want to allow executor committee members at the start of every epoch, but I guess the observer nodes should be allowed immediately?

Why `sig` and not `pk`, `id` or `nodeID`?